### DISCUSSION TITLE
starship install for Apline (iSH) on iOS

### DISCUSSION THREAD

**Discussion by jamesainslie** [starship/starship#1565] [Q&A] [answered]
I am using alpine (iSH) on my iPad, and love starship. I tried installing for bash.. but it fails with the following:

Configuration
> Bin directory: /usr/local/bin
> Platform:      unknown-linux-musl
> Arch:          i686

> Tarball URL: https://github.com/starship/starship/releases/latest/download/starship-i686-unknown-linux-musl.tar.gz
? Install Starship latest to /usr/local/bin? [y/N] y
! Bin directory /usr/local/bin is not in your $PATH
> Installing Starship, please wait…

x Command failed (exit code 22): curl --silent --fail --location https://github.com/starship/starship/releases/latest/download/starship-i686-unknown-linux-musl.tar.gz

> This is likely due to Starship not yet supporting your configuration.
> If you would like to see a build for your configuration,
> please create an issue requesting a build for i686-unknown-linux-musl:

**Comment by matchai** [comment, marked as answer]
Hey @jamesainslie 👋 
First off, thank you for sponsoring Starship! Your support is greatly appreciated. 🚀

As for getting an i686 musl build of Starship made, that should certainly be doable! I recently got an iPad, so I'll start looking into how we can get a working build of Starship for that platform.

**Reply by jamesainslie** [reply]
Epic! Let me know if you need help testing....
